docs/docs-components/Markdown.js

```diff
@@ -59,6 +60,22 @@ export default function Markdown({ textColor, text }: Props): Node {
       </h${level}>`;
   };
 
+  renderer.link = (href, title, children) =>
+    isPinchLink(href)
+      ? `<div style="display:inline;">
```

Comment:
Oh wacky, I assume this doesn't accept a JSX component?

Reply:
Nope. I'd remove this markdown framework and try to find a better one.